user-registration-app/src/app/views/user-registration-view/user-registration-view.component.html

```diff
@@ -71,48 +71,4 @@
     </div>
     <p class="center" style="margin-top: 20px;">Join the <a href="https://discord.gg/eyPSrNm9Cd">Discord here!</a></p>
   </li>
-  <li>
-    <!--<div class="collapsible-header">-->
-      <!--<i class="material-icons">card_travel</i>-->
-      <!--Business challenge registration-->
-      <!--<i class="material-icons right">keyboard_arrow_down</i>-->
-    <!--</div>-->
-    <!--<div class="collapsible-body">-->
-       <!--<a class="btn waves-effect waves-light submit-btn" (click)="businessChallengeRegister()">Register</a>-->
-    <!--</div>-->
```

Comment:
I think I accidentally removed the business challenge on the profile page

Reply:
that is indeed what this is. This is where the user has to go to sign up for the business challenge. don't know if we'll be having the business challenge again but this is probably worth keeping commented out in the file for now

Reply:
I feel like we probably won't(?) and I kinda just wanna nuke whatever we can for now, probably a good intro task too and we can come back to this PR later

Reply:
Should we just have like a file that has all our random html/css snippets we want to keep around just in case?

Reply:
I guess we could put it into the onenote